### PR TITLE
Fixed edge updates on one to one relationships

### PR DESCRIPTION
**Pull request by mhlz**
# Description

On 1to1 relationships updates for edge properties did not work in our project. I traced this back to a pretty simple fix in this library.

# Checklist

The following requirements should have been met (depending on the changes in the branch):

- [x] Integration tests have been updated
- [x] CLA (https://neo4j.com/developer/cla/) has been signed

### CODE REVIEW

**Comment by danstarns** [comment]
Hey, @mhlz. Thank you for your contribution. It doesn't look like the CLA has been signed. Could you follow the steps [here](https://neo4j.com/developer/cla/) making sure you use the same email as your GitHub commit?

**Comment by mhlz** [comment]
I sent it again from the correct email :)

File: packages/graphql/tests/integration/update.int.test.ts [hunk 1]
```diff
@@ -1322,6 +1322,98 @@ describe("update", () => {
...
+                        {
+                            test: true,
+                        },
+                    ],
+                },
+            });
+        } finally {
+            await session.close();
+        }
+    });
```
Comment (darrellwarde) [inline packages/graphql/tests/integration/update.int.test.ts:1415]: We generally put new integration tests which cover specific GitHub Issues in the `tests/integration/issues` directory (see existing examples in https://github.com/neo4j/graphql/tree/master/packages/graphql/tests/integration/issues).

Would you mind moving this into there? Thanks!

**Review by darrellwarde — CHANGES_REQUESTED** [review]
Would just like to see the integration test moved, but otherwise looks good!

File: packages/graphql/tests/integration/update.int.test.ts (hunk 1 above)
Comment (mhlz) [reply packages/graphql/tests/integration/update.int.test.ts]: I moved it :)

**Review by darrellwarde — APPROVED** [review]
LGTM, thanks! 🙂

**Pull request merged**